src/elements/OCAlert.vue

```diff
@@ -1,5 +1,5 @@
 <template>
-  <div class="oc-alert" :class="_class" uk-alert>
+  <div v-on:hide="onClose" class="oc-alert" :class="_class" uk-alert>
     <a v-if="!noClose" class="uk-alert-close" uk-close></a>
```

Comment:
maybe emit the event directly from this element and get rid of uk-close?

Reply:
In the future we could look at using oc-icon inside here, and emitting a close event. But styling is a mess at the moment trying to do that.